supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -16,28 +16,50 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
-
+When possible, rewrite documentation to avoid these terms:
 
-.Example: Removing 'blacklist'
-image:images/no.png[no] Heat blacklists any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+=== blacklist and whitelist
 
-image:images/yes.png[yes] Heat excludes any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
+.Example: Removing blacklist
+image:images/no.png[no] Heat _blacklists_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
+image:images/yes.png[yes] Heat _excludes_ any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.
 
-.Example: Removing 'whitelist'
-image:images/no.png[no] The following steps demonstrate adding a new rule to whitelist a custom binary.
+.Example: Removing whitelist
+image:images/no.png[no] The following steps demonstrate adding a new rule to _whitelist_ a custom binary.
 
-image:images/yes.png[yes] The following steps demonstrate adding a new rule to allow a custom binary.
+image:images/yes.png[yes] The following steps demonstrate adding a new rule to _allow_ a custom binary.
 
-When it is not possible to remove the terms 'blacklist' and 'whitelist', replace them with one of the following alternatives:
+When it is not possible to remove the terms _blacklist_ and _whitelist_, replace them with one of the following alternatives:
 
 * blocklist / allowlist: This combination is recommended by _The IBM Style Guide_. Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team.
 * denylist / allowlist
 * blocklist / passlist
 
+=== master and slave
+
+.Example: Removing _master_
+image:images/no.png[no] A Ceph Monitor maintains the _master_ copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+image:images/yes.png[yes] A Ceph Monitor maintains the _primary_ copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+image:images/yes.png[yes] A Ceph Monitor maintains the _main_ copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+
+.Example: Removing _slave_
+image:images/no.png[no] Copy the public key to the _slave_ node using the following command.
```

Comment:
Great addition to the guide here. My only comment is around the word 'using', which can produce a double translation when it's used like this. We could fix it by switching the example around:

no:
Use the following command to copy the public key to the _slave_ node.

yes:
Use the following command to copy the public key to the _secondary_ node.

Or we could add 'by' before 'using' to fix it also.

Reply:
Sounds good :)